just a few thousand commits

tashian opened the pull request:
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

PckgrBot committed:
New version: GitHub.cli version 2.39.2 (#128280)

wingetbot committed:
Automatic update of FireAlpaca.FireAlpaca from 2.11.12.0 to 2.11.13.0 (#128281)

* Creating manifests/f/FireAlpaca/FireAlpaca/2.11.13.0/FireAlpaca.FireAlpaca.installer.yaml

* Creating manifests/f/FireAlpaca/FireAlpaca/2.11.13.0/FireAlpaca.FireAlpaca.locale.en-US.yaml

* Creating manifests/f/FireAlpaca/FireAlpaca/2.11.13.0/FireAlpaca.FireAlpaca.yaml

* Updating manifests/f/FireAlpaca/FireAlpaca/2.11.13.0/FireAlpaca.FireAlpaca.installer.yaml

* Updating manifests/f/FireAlpaca/FireAlpaca/2.11.13.0/FireAlpaca.FireAlpaca.locale.en-US.yaml

* Updating manifests/f/FireAlpaca/FireAlpaca/2.11.13.0/FireAlpaca.FireAlpaca.yaml

wingetbot committed:
Automatic update of AxCrypt.AxCrypt from 2.1.1636.0 to 2.1.1647.0 (#128283)

* Creating manifests/a/AxCrypt/AxCrypt/2.1.1647.0/AxCrypt.AxCrypt.installer.yaml

* Creating manifests/a/AxCrypt/AxCrypt/2.1.1647.0/AxCrypt.AxCrypt.locale.en-US.yaml

* Creating manifests/a/AxCrypt/AxCrypt/2.1.1647.0/AxCrypt.AxCrypt.yaml

* Updating manifests/a/AxCrypt/AxCrypt/2.1.1647.0/AxCrypt.AxCrypt.installer.yaml

* Updating manifests/a/AxCrypt/AxCrypt/2.1.1647.0/AxCrypt.AxCrypt.locale.en-US.yaml

* Updating manifests/a/AxCrypt/AxCrypt/2.1.1647.0/AxCrypt.AxCrypt.yaml

wingetbot committed:
Automatic update of Google.ChromeRemoteDesktop from 119.0.6045.21 to 120.0.6099.8 (#128286)

* Creating manifests/g/Google/ChromeRemoteDesktop/120.0.6099.8/Google.ChromeRemoteDesktop.installer.yaml

* Creating manifests/g/Google/ChromeRemoteDesktop/120.0.6099.8/Google.ChromeRemoteDesktop.locale.en-US.yaml

* Creating manifests/g/Google/ChromeRemoteDesktop/120.0.6099.8/Google.ChromeRemoteDesktop.yaml

* Updating manifests/g/Google/ChromeRemoteDesktop/120.0.6099.8/Google.ChromeRemoteDesktop.installer.yaml

* Updating manifests/g/Google/ChromeRemoteDesktop/120.0.6099.8/Google.ChromeRemoteDesktop.locale.en-US.yaml

* Updating manifests/g/Google/ChromeRemoteDesktop/120.0.6099.8/Google.ChromeRemoteDesktop.yaml

wingetbot committed:
Automatic update of BitSum.ProcessLasso.Beta from 12.4.3.7 to 12.4.3.9 (#128288)

* Creating manifests/b/BitSum/ProcessLasso/Beta/12.4.3.9/BitSum.ProcessLasso.Beta.installer.yaml

* Creating manifests/b/BitSum/ProcessLasso/Beta/12.4.3.9/BitSum.ProcessLasso.Beta.locale.en-US.yaml

* Creating manifests/b/BitSum/ProcessLasso/Beta/12.4.3.9/BitSum.ProcessLasso.Beta.yaml

* Updating manifests/b/BitSum/ProcessLasso/Beta/12.4.3.9/BitSum.ProcessLasso.Beta.installer.yaml

* Updating manifests/b/BitSum/ProcessLasso/Beta/12.4.3.9/BitSum.ProcessLasso.Beta.locale.en-US.yaml

* Updating manifests/b/BitSum/ProcessLasso/Beta/12.4.3.9/BitSum.ProcessLasso.Beta.yaml

SpecterShell committed:
New version: Brave.Brave.Nightly version 119.1.62.84 (#128289)

SpecterShell committed:
New version: Figma.Figma version 116.15.4 (#128291)

SpecterShell committed:
New version: RedHat.Podman version 4.8.0 (#128292)

SpecterShell committed:
New version: TeamIDE.TeamIDE version 2.5.5 (#128293)

SpecterShell committed:
New version: DuckDuckGo.DesktopBrowser version 0.59.0.0 (#128294)

SpecterShell committed:
New version: Baidu.BaiduNetdisk version 7.35.0 (#128296)

SpecterShell committed:
New version: Kingsoft.WPSOffice.CN version 12.1.0.15990 (#128297)

fxbf committed:
New version: NitrokeyGmbH.Nitropy version 0.4.42 (#128298)

SpecterShell committed:
New version: mikf.gallery-dl version 1.26.3 (#128299)

PckgrBot committed:
New version: WinMerge.WinMerge version 2.16.36 (#128300)

wingetbot committed:
Automatic update of Neovim.Neovim.Nightly 0.10.0 (#128301)

* Updating manifests/n/Neovim/Neovim/Nightly/0.10.0/Neovim.Neovim.Nightly.installer.yaml

* Updating manifests/n/Neovim/Neovim/Nightly/0.10.0/Neovim.Neovim.Nightly.locale.en-US.yaml

* Updating manifests/n/Neovim/Neovim/Nightly/0.10.0/Neovim.Neovim.Nightly.yaml

wingetbot committed:
Automatic update of Canva.Canva from 1.77.0 to 1.78.0 (#128302)

* Creating manifests/c/Canva/Canva/1.78.0/Canva.Canva.installer.yaml

* Creating manifests/c/Canva/Canva/1.78.0/Canva.Canva.locale.en-US.yaml

* Creating manifests/c/Canva/Canva/1.78.0/Canva.Canva.yaml

* Updating manifests/c/Canva/Canva/1.78.0/Canva.Canva.installer.yaml

* Updating manifests/c/Canva/Canva/1.78.0/Canva.Canva.locale.en-US.yaml

* Updating manifests/c/Canva/Canva/1.78.0/Canva.Canva.yaml

skycommand committed:
Update Abdelrhman-AK.WinPaletter.installer.yaml – Fix critical typo (#128306)

Same problem as #128150: This package (Abdelrhman-AK.WinPaletter) incorrectly registers "zip" as its portable command alias. Given the nature of the package, this egregious registration could only be a typo.

rsteube-bot committed:
Remove version: rsteube.Carapace version 0.25.2 (#128311)

PckgrBot committed:
New version: SiberSystems.RoboForm version 9.5.5 (#128307)

LeonGungadinMogensen committed:
New version: Toit.Jaguar version 1.22.0 (#128054)

dobsontom committed:
Update version: Microsoft.PowerBI version 2.123.742.0 (#128274)

* Update version: Microsoft.PowerBI version 2.123.742.0

* Update version: Microsoft.PowerBI version 2.123.742.0

wingetbot committed:
Automatic update of Evope.Evope from 1.1.4.53 to 1.1.4.55 (#128277)

* Creating manifests/e/Evope/Evope/1.1.4.55/Evope.Evope.installer.yaml

* Creating manifests/e/Evope/Evope/1.1.4.55/Evope.Evope.locale.pt-BR.yaml

* Creating manifests/e/Evope/Evope/1.1.4.55/Evope.Evope.yaml

* Updating manifests/e/Evope/Evope/1.1.4.55/Evope.Evope.installer.yaml

* Updating manifests/e/Evope/Evope/1.1.4.55/Evope.Evope.locale.pt-BR.yaml

* Updating manifests/e/Evope/Evope/1.1.4.55/Evope.Evope.yaml

wingetbot committed:
Automatic deletion of Canva.Canva 1.77.0 (#128303)

* Updating manifests/c/Canva/Canva/1.77.0/Canva.Canva.installer.yaml

* Updating manifests/c/Canva/Canva/1.77.0/Canva.Canva.locale.en-US.yaml

* Updating manifests/c/Canva/Canva/1.77.0/Canva.Canva.yaml

wingetbot committed:
Automatic update of EvgenyCherpak.RemoteHelperApp from 2023.1 to 2023.5 (#128278)

* Creating manifests/e/EvgenyCherpak/RemoteHelperApp/2023.5/EvgenyCherpak.RemoteHelperApp.installer.yaml

* Creating manifests/e/EvgenyCherpak/RemoteHelperApp/2023.5/EvgenyCherpak.RemoteHelperApp.locale.en-US.yaml

* Creating manifests/e/EvgenyCherpak/RemoteHelperApp/2023.5/EvgenyCherpak.RemoteHelperApp.yaml

* Updating manifests/e/EvgenyCherpak/RemoteHelperApp/2023.5/EvgenyCherpak.RemoteHelperApp.installer.yaml

* Updating manifests/e/EvgenyCherpak/RemoteHelperApp/2023.5/EvgenyCherpak.RemoteHelperApp.locale.en-US.yaml

* Updating manifests/e/EvgenyCherpak/RemoteHelperApp/2023.5/EvgenyCherpak.RemoteHelperApp.yaml

wingetbot committed:
Automatic deletion of DirSoft.ARPro 17.0.1.1086 (#128315)

* Updating manifests/d/DirSoft/ARPro/17.0.1.1086/DirSoft.ARPro.installer.yaml

* Updating manifests/d/DirSoft/ARPro/17.0.1.1086/DirSoft.ARPro.locale.en-US.yaml

* Updating manifests/d/DirSoft/ARPro/17.0.1.1086/DirSoft.ARPro.yaml

wingetbot committed:
Automatic deletion of PIKCLOUD.PikPak 2.1.4 (#128317)

* Updating manifests/p/PIKCLOUD/PikPak/2.1.4/PIKCLOUD.PikPak.installer.yaml

* Updating manifests/p/PIKCLOUD/PikPak/2.1.4/PIKCLOUD.PikPak.locale.en-US.yaml

* Updating manifests/p/PIKCLOUD/PikPak/2.1.4/PIKCLOUD.PikPak.locale.zh-CN.yaml

* Updating manifests/p/PIKCLOUD/PikPak/2.1.4/PIKCLOUD.PikPak.yaml

SpecterShell committed:
New version: NetEase.YoudaoPokeClass version 2.11.4 (#128295)

nvim-winget committed:
Update version: Neovim.Neovim.Nightly version 0.10.0 (#128334)

wingetbot committed:
Automatic deletion of Zoom.ZoomOutlookPlugin 5.15.5 (#128330)

* Updating manifests/z/Zoom/ZoomOutlookPlugin/5.15.5/Zoom.ZoomOutlookPlugin.installer.yaml

* Updating manifests/z/Zoom/ZoomOutlookPlugin/5.15.5/Zoom.ZoomOutlookPlugin.locale.en-US.yaml

* Updating manifests/z/Zoom/ZoomOutlookPlugin/5.15.5/Zoom.ZoomOutlookPlugin.locale.zh-CN.yaml

* Updating manifests/z/Zoom/ZoomOutlookPlugin/5.15.5/Zoom.ZoomOutlookPlugin.yaml

wingetbot committed:
Automatic deletion of XnSoft.XnViewMP 1.6.1.0 (#128324)

* Updating manifests/x/XnSoft/XnViewMP/1.6.1.0/XnSoft.XnViewMP.installer.yaml

* Updating manifests/x/XnSoft/XnViewMP/1.6.1.0/XnSoft.XnViewMP.locale.en-US.yaml

* Updating manifests/x/XnSoft/XnViewMP/1.6.1.0/XnSoft.XnViewMP.yaml

wingetbot committed:
Automatic deletion of WilhelmHappe.NetRadio 2.5.2.2 (#128321)

* Updating manifests/w/WilhelmHappe/NetRadio/2.5.2.2/WilhelmHappe.NetRadio.installer.yaml

* Updating manifests/w/WilhelmHappe/NetRadio/2.5.2.2/WilhelmHappe.NetRadio.locale.en-US.yaml

* Updating manifests/w/WilhelmHappe/NetRadio/2.5.2.2/WilhelmHappe.NetRadio.yaml

SpecterShell committed:
Update: vivo.OfficeKit version 3.1.3.0 (#128344)

SpecterShell committed:
New version: Brave.Brave.Beta version 119.1.61.91 (#128343)

SpecterShell committed:
New version: Bilibili.Livehime version 4.56.0.6131 (#128341)

SpecterShell committed:
New version: B3log.SiYuan version 2.11.0 (#128338)

SpecterShell committed:
New version: Glodon.CADReader.CN version 6.0.1.94 (#128342)

SpecterShell committed:
New version: Microsoft.EdgeDriver version 119.0.2151.93 (#128337)

wingetbot committed:
Automatic update of DucFabulous.UltraViewer from 6.6.59 to 6.6.61 (#128318)

* Creating manifests/d/DucFabulous/UltraViewer/6.6.61/DucFabulous.UltraViewer.installer.yaml

* Creating manifests/d/DucFabulous/UltraViewer/6.6.61/DucFabulous.UltraViewer.locale.en-US.yaml

* Creating manifests/d/DucFabulous/UltraViewer/6.6.61/DucFabulous.UltraViewer.yaml

* Updating manifests/d/DucFabulous/UltraViewer/6.6.61/DucFabulous.UltraViewer.installer.yaml

* Updating manifests/d/DucFabulous/UltraViewer/6.6.61/DucFabulous.UltraViewer.locale.en-US.yaml

* Updating manifests/d/DucFabulous/UltraViewer/6.6.61/DucFabulous.UltraViewer.yaml

wingetbot committed:
Automatic deletion of DucFabulous.UltraViewer 6.6.59 (#128319)

* Updating manifests/d/DucFabulous/UltraViewer/6.6.59/DucFabulous.UltraViewer.installer.yaml

* Updating manifests/d/DucFabulous/UltraViewer/6.6.59/DucFabulous.UltraViewer.locale.en-US.yaml

* Updating manifests/d/DucFabulous/UltraViewer/6.6.59/DucFabulous.UltraViewer.yaml

wingetbot committed:
Automatic update of TweakingTechnologies.TweakShotScreenCapture from 1.0.1.40908 to 1.0.23.11103 (#128312)

* Creating manifests/t/TweakingTechnologies/TweakShotScreenCapture/1.0.23.11103/TweakingTechnologies.TweakShotScreenCapture.installer.yaml

* Creating manifests/t/TweakingTechnologies/TweakShotScreenCapture/1.0.23.11103/TweakingTechnologies.TweakShotScreenCapture.locale.en-US.yaml

* Creating manifests/t/TweakingTechnologies/TweakShotScreenCapture/1.0.23.11103/TweakingTechnologies.TweakShotScreenCapture.yaml

* Updating manifests/t/TweakingTechnologies/TweakShotScreenCapture/1.0.23.11103/TweakingTechnologies.TweakShotScreenCapture.installer.yaml

* Updating manifests/t/TweakingTechnologies/TweakShotScreenCapture/1.0.23.11103/TweakingTechnologies.TweakShotScreenCapture.locale.en-US.yaml

* Updating manifests/t/TweakingTechnologies/TweakShotScreenCapture/1.0.23.11103/TweakingTechnologies.TweakShotScreenCapture.yaml

Dragon1573 committed:
[New version]: Tonec.InternetDownloadManager version 6.42.2 (#128351)

* Fix (Tonec.InternetDownloadManager): Remove whitespaces in PackageVersion

Signed-off-by: Dragon1573 <49941141+Dragon1573@users.noreply.github.com>

* Remove version: Tonec.InternetDownloadManager version 6.42.1

Signed-off-by: Dragon1573 <49941141+Dragon1573@users.noreply.github.com>

* Revert "Remove version: Tonec.InternetDownloadManager version 6.42.1"

This reverts commit 071d27accc748ad9d32115b706fe333b56e5d61d.

Signed-off-by: Dragon1573 <49941141+Dragon1573@users.noreply.github.com>

---------

Signed-off-by: Dragon1573 <49941141+Dragon1573@users.noreply.github.com>

Dragon1573 committed:
Remove version: Tonec.InternetDownloadManager version 6.42.1 (#128383)

Signed-off-by: Dragon1573 <49941141+Dragon1573@users.noreply.github.com>

TotTheMax committed:
Submitting Studio-CLI version 23.9.1 (#128234)

Co-authored-by: Xiaoyu Jiang <Xiaoyu.Jiang@windriver.com>

Commit:
New version: RiotGames.LeagueOfLegends.BR version 64.0.10.5036631 (#128384)

* New version: RiotGames.LeagueOfLegends.BR version 64.0.10.5036631

* Delete manifests/r/RiotGames/LeagueOfLegends/BR/11.13.382.1241/RiotGames.LeagueOfLegends.BR.installer.yaml

* Delete manifests/r/RiotGames/LeagueOfLegends/BR/11.13.382.1241/RiotGames.LeagueOfLegends.BR.locale.en-US.yaml

* Delete manifests/r/RiotGames/LeagueOfLegends/BR/11.13.382.1241/RiotGames.LeagueOfLegends.BR.yaml

Trenly committed:
Remove: KCSoftwares.ApHeMo version 2.20.0.31 (#128402)

Trenly committed:
Remove: KCSoftwares.BATExpert version 1.14.5.27 (#128401)

Trenly committed:
Remove: KCSoftwares.dot11Expert version 1.6.3.18 (#128400)

Trenly committed:
Remove: KCSoftwares.DUMo version 2.25.4.125 (#128399)

Trenly committed:
Remove: KCSoftwares.HDDExpert version 1.20.1.55 (#128398)

Trenly committed:
Remove: KCSoftwares.IDPhotoStudio version 2.16.5.75 (#128397)

Trenly committed:
Remove: KCSoftwares.KCleaner version 3.8.6.116 (#128396)

Trenly committed:
Remove: KCSoftwares.MassCert version 2.1.1.20 (#128395)

Trenly committed:
Remove: KCSoftwares.PhotoToFilm version 3.9.8.107 (#128394)

Trenly committed:
Remove: KCSoftwares.PortExpert version 1.8.4.23 (#128393)

Trenly committed:
Remove: KCSoftwares.VideoInspector version 2.15.10.154 (#128391)

Trenly committed:
Remove: KCSoftwares.SUMo version 5.17.10.542 (#128390)

Trenly committed:
Remove: KCSoftwares.RAMExpert version 1.23.0.47 (#128392)

LeonarddeR committed:
New version: Cockos.REAPER version 7.06 (#128304)

wingetbot committed:
Automatic update of OxygenCloud.odrive from 1.0.7226 to 1.0.7231 (#128309)

* Creating manifests/o/OxygenCloud/odrive/1.0.7231/OxygenCloud.odrive.installer.yaml

* Creating manifests/o/OxygenCloud/odrive/1.0.7231/OxygenCloud.odrive.locale.en-US.yaml

* Creating manifests/o/OxygenCloud/odrive/1.0.7231/OxygenCloud.odrive.yaml

* Updating manifests/o/OxygenCloud/odrive/1.0.7231/OxygenCloud.odrive.installer.yaml

* Updating manifests/o/OxygenCloud/odrive/1.0.7231/OxygenCloud.odrive.locale.en-US.yaml

* Updating manifests/o/OxygenCloud/odrive/1.0.7231/OxygenCloud.odrive.yaml

wingetbot committed:
Automatic update of Ubisoft.Connect from 147.0.10965 to 148.1.0.10975 (#128314)

* Creating manifests/u/Ubisoft/Connect/148.1.0.10975/Ubisoft.Connect.installer.yaml

* Creating manifests/u/Ubisoft/Connect/148.1.0.10975/Ubisoft.Connect.locale.en-US.yaml

* Creating manifests/u/Ubisoft/Connect/148.1.0.10975/Ubisoft.Connect.yaml

* Updating manifests/u/Ubisoft/Connect/148.1.0.10975/Ubisoft.Connect.installer.yaml

* Updating manifests/u/Ubisoft/Connect/148.1.0.10975/Ubisoft.Connect.locale.en-US.yaml

* Updating manifests/u/Ubisoft/Connect/148.1.0.10975/Ubisoft.Connect.yaml

wingetbot committed:
Automatic update of LabyMediaGmbH.LabyModLauncher from 2.0.1 to 2.0.2 (#128320)

* Creating manifests/l/LabyMediaGmbH/LabyModLauncher/2.0.2/LabyMediaGmbH.LabyModLauncher.installer.yaml

* Creating manifests/l/LabyMediaGmbH/LabyModLauncher/2.0.2/LabyMediaGmbH.LabyModLauncher.locale.en-US.yaml

* Creating manifests/l/LabyMediaGmbH/LabyModLauncher/2.0.2/LabyMediaGmbH.LabyModLauncher.yaml

* Updating manifests/l/LabyMediaGmbH/LabyModLauncher/2.0.2/LabyMediaGmbH.LabyModLauncher.installer.yaml

* Updating manifests/l/LabyMediaGmbH/LabyModLauncher/2.0.2/LabyMediaGmbH.LabyModLauncher.locale.en-US.yaml

* Updating manifests/l/LabyMediaGmbH/LabyModLauncher/2.0.2/LabyMediaGmbH.LabyModLauncher.yaml

PckgrBot committed:
New version: Amazon.AWSCLI version 2.14.0 (#128328)

wingetbot committed:
Automatic deletion of Salesforce.sfdx-cli 7.201.6 (#128329)

* Updating manifests/s/Salesforce/sfdx-cli/7.201.6/Salesforce.sfdx-cli.installer.yaml

* Updating manifests/s/Salesforce/sfdx-cli/7.201.6/Salesforce.sfdx-cli.locale.en-US.yaml

* Updating manifests/s/Salesforce/sfdx-cli/7.201.6/Salesforce.sfdx-cli.yaml

blacktop committed:
New version: blacktop.ipsw 3.1.425 (#128335)

* New version: blacktop.ipsw 3.1.425: add version

* New version: blacktop.ipsw 3.1.425: add installer

* New version: blacktop.ipsw 3.1.425: add locale

---------

Co-authored-by: goreleaserbot <bot@goreleaser.com>

MesterPerfect committed:
New version: tecwindow.SoundTranscriber version 1.3.1 (#128332)

* New version: tecwindow.SoundTranscriber version 1.3.1

* Update tecwindow.SoundTranscriber.installer.yaml fixed

* Update tecwindow.SoundTranscriber.locale.en-US.yaml fixed

* Update tecwindow.SoundTranscriber.yaml fixed

kbdharun committed:
New version: gaphor.gaphor 2.22.0 (#128333)

russellbanks committed:
New version: StandardNotes.StandardNotes version 3.181.33 (#128345)

blacktop committed:
New version: blacktop.ipswd 3.1.425 (#128336)

* New version: blacktop.ipswd 3.1.425: add version

* New version: blacktop.ipswd 3.1.425: add installer

* New version: blacktop.ipswd 3.1.425: add locale

---------

Co-authored-by: goreleaserbot <bot@goreleaser.com>

Commit:
New package: WHONET.2023 version 23.11.27 (#127890)

* New package: WHONET.2023 version 23.11.16

* Update WHONET.2023.locale.en-US.yaml

* Update WHONET.2023.locale.en-US.yaml

* Update WHONET.2023.locale.en-US.yaml

* Update WHONET.2023.locale.en-US.yaml

* Update WHONET.2023.locale.en-US.yaml

* New version

russellbanks committed:
New version: StandardNotes.StandardNotes version 3.181.34 (#128346)

russellbanks committed:
New version: StandardNotes.StandardNotes version 3.181.35 (#128347)

russellbanks committed:
New version: StandardNotes.StandardNotes version 3.181.36 (#128348)

russellbanks committed:
New version: StandardNotes.StandardNotes version 3.181.37 (#128349)

MrHinsh committed:
New version: nkdAgility.AzureDevOpsMigrationTools version 14.3.3 (#128353)

russellbanks committed:
New version: StandardNotes.StandardNotes version 3.181.38 (#128350)

BinBinTsai committed:
New version: Splashtop.SplashtopStreamer version 3.6.2.0 (#128357)

BinBinTsai committed:
New version: Splashtop.SplashtopStreamer.Deployment version 3.6.2.0 (#128355)

BinBinTsai committed:
New version: Splashtop.SOS version 3.6.2.0 (#128358)

PckgrBot committed:
New version: Amazon.AWSCLI version 2.14.1 (#128359)

russellbanks committed:
New version: EclipseAdoptium.Temurin.11.JDK version 11.0.21.9 (#128360)

russellbanks committed:
New version: EclipseAdoptium.Temurin.11.JRE version 11.0.21.9 (#128361)

xpouyat committed:
Microsoft.AzureMediaServicesExplorer version 5.8.0 (#128364)

russellbanks committed:
New version: EclipseAdoptium.Temurin.8.JDK version 8.0.392.8 (#128365)

mbialon committed:
New version: spacelift-io.spacectl 0.28.0 (#128367)

* New version: spacelift-io.spacectl 0.28.0: add version

* New version: spacelift-io.spacectl 0.28.0: add installer

* New version: spacelift-io.spacectl 0.28.0: add locale

---------

Co-authored-by: goreleaserbot <bot@goreleaser.com>

russellbanks committed:
New version: EclipseAdoptium.Temurin.8.JRE version 8.0.392.8 (#128366)

russellbanks committed:
New version: EclipseAdoptium.Temurin.17.JDK version 17.0.9.9 (#128368)

russellbanks committed:
New version: EclipseAdoptium.Temurin.17.JRE version 17.0.9.9 (#128369)

CHIZI-0618 committed:
SagerNet.sing-box version 1.6.7 (#128372)

CennoxX committed:
New version: BurntSushi.ripgrep.MSVC version 14.0.2 (#128388)

jay-51 committed:
New package: RiotGames.LeagueOfLegends.JP version 64.0.10.5036631 (#128363)

jay-51 committed:
New package: RiotGames.LeagueOfLegends.TR version 64.0.10.5036631 (#128371)

sansmoraxz committed:
New package: Docker.DockerCompose version 2.23.3 (#128386)

jay-51 committed:
New package: RiotGames.LeagueOfLegends.RU version 64.0.10.5036631 (#128370)

jay-51 committed:
New package: RiotGames.LeagueOfLegends.KR version 64.0.10.5036631 (#128373)

jay-51 committed:
New package: RiotGames.LeagueOfLegends.SG2 version 64.0.10.5036631 (#128374)

Trenly committed:
Add AppsAndFeaturesEntries for DosBox (#128403)

jay-51 committed:
New package: RiotGames.LeagueOfLegends.PH2 version 64.0.10.5036631 (#128375)

jay-51 committed:
New package: RiotGames.LeagueOfLegends.TH2 version 64.0.10.5036631 (#128377)

jay-51 committed:
Update AsiasoftOnline.PlayparkDownloader.locale.en-US.yaml (#128411)

xiSage committed:
New version: YuzuEmu.Yuzu.Mainline version 1635 (#128415)

doppler-bot committed:
New version: Doppler.doppler 3.66.4 (#128414)

* New version: Doppler.doppler 3.66.4: add version

* New version: Doppler.doppler 3.66.4: add installer

* New version: Doppler.doppler 3.66.4: add locale

CennoxX committed:
New version: BurntSushi.ripgrep.GNU version 14.0.2 (#128387)

rsteube-bot committed:
New version: rsteube.Carapace version 0.28.4 (#128310)

chrisant996 committed:
New version: chrisant996.Clink version 1.5.18 (#128339)

Dvd-Znf committed:
New package: KDE.Filelight (#128410)

jay-51 committed:
New package: RiotGames.LeagueOfLegends.TW2 version 64.0.10.5036631 (#128378)

jay-51 committed:
New package: RiotGames.LeagueOfLegends.VN2 version 64.0.10.5036631 (#128376)

wingetbot committed:
Automatic deletion of AgileBits.1Password.Beta 8.10.22-1.BETA (#128428)

* Updating manifests/a/AgileBits/1Password/Beta/8.10.22-1.BETA/AgileBits.1Password.Beta.installer.yaml

* Updating manifests/a/AgileBits/1Password/Beta/8.10.22-1.BETA/AgileBits.1Password.Beta.locale.en-US.yaml

* Updating manifests/a/AgileBits/1Password/Beta/8.10.22-1.BETA/AgileBits.1Password.Beta.yaml

PckgrBot committed:
New version: Google.Chrome version 119.0.6045.200 (#128437)

caarlos0 committed:
New version: caarlos0.fork-cleaner 2.3.1 (#126784)

* New version: caarlos0.fork-cleaner 2.3.1: add version

* New version: caarlos0.fork-cleaner 2.3.1: add installer

* New version: caarlos0.fork-cleaner 2.3.1: add locale

---------

Co-authored-by: goreleaserbot <bot@goreleaser.com>

Dragon1573 committed:
New version: Unity.Unity.2022 version 2022.3.13f1 (#127308)

* New version Unity.Unity.2022 version 2022.3.13f1

Signed-off-by: GitHub <noreply@github.com>

* Update (installer.yaml): ElevationRequirement

- Unity is not designed to run as Administrator, elevation behaviour should be prohibited.

Signed-off-by: Dragon1573 <49941141+Dragon1573@users.noreply.github.com>

* Revert "Update (installer.yaml): ElevationRequirement"

This reverts commit 78bedf37828ac28c2a96fcc89b503c50f9870379.

---------

Signed-off-by: GitHub <noreply@github.com>
Signed-off-by: Dragon1573 <49941141+Dragon1573@users.noreply.github.com>

PckgrBot committed:
New version: Hashicorp.Vagrant version 2.4.0 (#127962)

PckgrBot committed:
New version: JetBrains.TeamCity version 2023.11 (#128409)

atilluF committed:
New version: ApacheFriends.Xampp.8.2 version 8.2.12 (#128412)

edwardskrod committed:
Microsoft.VisualStudio.2022.Enterprise version 17.8.2 (#128418)

edwardskrod committed:
Microsoft.VisualStudio.2022.Community version 17.8.2 (#128419)

edwardskrod committed:
Microsoft.VisualStudio.2022.Professional version 17.8.2 (#128420)

edwardskrod committed:
Microsoft.VisualStudio.2022.BuildTools version 17.8.2 (#128421)

edwardskrod committed:
Microsoft.VisualStudio.2022.Enterprise.Preview version 17.9.0 (#128423)

edwardskrod committed:
Microsoft.VisualStudio.2022.Professional.Preview version 17.9.0 (#128424)

edwardskrod committed:
Microsoft.VisualStudio.2022.Community.Preview version 17.9.0 (#128425)

wingetbot committed:
Automatic update of ESET.EndpointAntivirus from 10.1.2058.0 to 11.0.2032.0 (#128430)

* Creating manifests/e/ESET/EndpointAntivirus/11.0.2032.0/ESET.EndpointAntivirus.installer.yaml

* Creating manifests/e/ESET/EndpointAntivirus/11.0.2032.0/ESET.EndpointAntivirus.locale.en-US.yaml

* Creating manifests/e/ESET/EndpointAntivirus/11.0.2032.0/ESET.EndpointAntivirus.yaml

* Updating manifests/e/ESET/EndpointAntivirus/11.0.2032.0/ESET.EndpointAntivirus.installer.yaml

* Updating manifests/e/ESET/EndpointAntivirus/11.0.2032.0/ESET.EndpointAntivirus.locale.en-US.yaml

* Updating manifests/e/ESET/EndpointAntivirus/11.0.2032.0/ESET.EndpointAntivirus.yaml

wingetbot committed:
Automatic deletion of EvgenyCherpak.RemoteHelperApp 2023.1 (#128431)

* Updating manifests/e/EvgenyCherpak/RemoteHelperApp/2023.1/EvgenyCherpak.RemoteHelperApp.installer.yaml

* Updating manifests/e/EvgenyCherpak/RemoteHelperApp/2023.1/EvgenyCherpak.RemoteHelperApp.locale.en-US.yaml

* Updating manifests/e/EvgenyCherpak/RemoteHelperApp/2023.1/EvgenyCherpak.RemoteHelperApp.yaml

wingetbot committed:
Automatic deletion of Evope.Evope 1.1.4.53 (#128432)

* Updating manifests/e/Evope/Evope/1.1.4.53/Evope.Evope.installer.yaml

* Updating manifests/e/Evope/Evope/1.1.4.53/Evope.Evope.locale.pt-BR.yaml

* Updating manifests/e/Evope/Evope/1.1.4.53/Evope.Evope.yaml

wingetbot committed:
Automatic deletion of Facebook.Messenger 196.0.518282206 (#128433)

* Updating manifests/f/Facebook/Messenger/196.0.518282206/Facebook.Messenger.installer.yaml

* Updating manifests/f/Facebook/Messenger/196.0.518282206/Facebook.Messenger.locale.en-US.yaml

* Updating manifests/f/Facebook/Messenger/196.0.518282206/Facebook.Messenger.yaml

wingetbot committed:
Automatic update of FireAlpaca.FireAlpaca from 2.11.12.0 to 2.11.14.0 (#128434)

* Creating manifests/f/FireAlpaca/FireAlpaca/2.11.14.0/FireAlpaca.FireAlpaca.installer.yaml

* Creating manifests/f/FireAlpaca/FireAlpaca/2.11.14.0/FireAlpaca.FireAlpaca.locale.en-US.yaml

* Creating manifests/f/FireAlpaca/FireAlpaca/2.11.14.0/FireAlpaca.FireAlpaca.yaml

* Updating manifests/f/FireAlpaca/FireAlpaca/2.11.14.0/FireAlpaca.FireAlpaca.installer.yaml

* Updating manifests/f/FireAlpaca/FireAlpaca/2.11.14.0/FireAlpaca.FireAlpaca.locale.en-US.yaml

* Updating manifests/f/FireAlpaca/FireAlpaca/2.11.14.0/FireAlpaca.FireAlpaca.yaml

wingetbot committed:
Automatic deletion of FireAlpaca.FireAlpaca 2.11.12.0 (#128435)

* Updating manifests/f/FireAlpaca/FireAlpaca/2.11.12.0/FireAlpaca.FireAlpaca.installer.yaml

* Updating manifests/f/FireAlpaca/FireAlpaca/2.11.12.0/FireAlpaca.FireAlpaca.locale.en-US.yaml

* Updating manifests/f/FireAlpaca/FireAlpaca/2.11.12.0/FireAlpaca.FireAlpaca.yaml

wingetbot committed:
Automatic deletion of FireAlpaca.FireAlpaca 2.11.13.0 (#128439)

* Updating manifests/f/FireAlpaca/FireAlpaca/2.11.13.0/FireAlpaca.FireAlpaca.installer.yaml

* Updating manifests/f/FireAlpaca/FireAlpaca/2.11.13.0/FireAlpaca.FireAlpaca.locale.en-US.yaml

* Updating manifests/f/FireAlpaca/FireAlpaca/2.11.13.0/FireAlpaca.FireAlpaca.yaml

babblebauble committed:
Remove: Morpheus.Bazarr version 1.2.0 (#128438)

wingetbot committed:
Automatic deletion of AxCrypt.AxCrypt 2.1.1636.0 (#128440)

* Updating manifests/a/AxCrypt/AxCrypt/2.1.1636.0/AxCrypt.AxCrypt.installer.yaml

* Updating manifests/a/AxCrypt/AxCrypt/2.1.1636.0/AxCrypt.AxCrypt.locale.en-US.yaml

* Updating manifests/a/AxCrypt/AxCrypt/2.1.1636.0/AxCrypt.AxCrypt.yaml

wingetbot committed:
Automatic deletion of SiberSystems.GoodSync 12.4.6.6 (#128443)

* Updating manifests/s/SiberSystems/GoodSync/12.4.6.6/SiberSystems.GoodSync.installer.yaml

* Updating manifests/s/SiberSystems/GoodSync/12.4.6.6/SiberSystems.GoodSync.locale.en-US.yaml

* Updating manifests/s/SiberSystems/GoodSync/12.4.6.6/SiberSystems.GoodSync.yaml

stacklokbot committed:
New version: stacklok.minder 0.0.19 (#128441)

* New version: stacklok.minder 0.0.19: add version

* New version: stacklok.minder 0.0.19: add installer

* New version: stacklok.minder 0.0.19: add locale

---------

Co-authored-by: stacklokbot <info@stacklok.com>

wingetbot committed:
Automatic deletion of SiberSystems.GoodSync 12.4.7.7 (#128445)

* Updating manifests/s/SiberSystems/GoodSync/12.4.7.7/SiberSystems.GoodSync.installer.yaml

* Updating manifests/s/SiberSystems/GoodSync/12.4.7.7/SiberSystems.GoodSync.locale.en-US.yaml

* Updating manifests/s/SiberSystems/GoodSync/12.4.7.7/SiberSystems.GoodSync.yaml

wingetbot committed:
Automatic deletion of SiberSystems.RoboForm 9.5.4 (#128444)

* Updating manifests/s/SiberSystems/RoboForm/9.5.4/SiberSystems.RoboForm.installer.yaml

* Updating manifests/s/SiberSystems/RoboForm/9.5.4/SiberSystems.RoboForm.locale.en-US.yaml

* Updating manifests/s/SiberSystems/RoboForm/9.5.4/SiberSystems.RoboForm.yaml

wingetbot committed:
Automatic deletion of Google.ChromeRemoteDesktop 119.0.6045.21 (#128446)

* Updating manifests/g/Google/ChromeRemoteDesktop/119.0.6045.21/Google.ChromeRemoteDesktop.installer.yaml

* Updating manifests/g/Google/ChromeRemoteDesktop/119.0.6045.21/Google.ChromeRemoteDesktop.locale.en-US.yaml

* Updating manifests/g/Google/ChromeRemoteDesktop/119.0.6045.21/Google.ChromeRemoteDesktop.yaml

wingetbot committed:
Automatic update of BitSum.ProcessLasso.Beta from 12.4.3.7 to 12.4.3.13 (#128448)

* Creating manifests/b/BitSum/ProcessLasso/Beta/12.4.3.13/BitSum.ProcessLasso.Beta.installer.yaml

* Creating manifests/b/BitSum/ProcessLasso/Beta/12.4.3.13/BitSum.ProcessLasso.Beta.locale.en-US.yaml

* Creating manifests/b/BitSum/ProcessLasso/Beta/12.4.3.13/BitSum.ProcessLasso.Beta.yaml

* Updating manifests/b/BitSum/ProcessLasso/Beta/12.4.3.13/BitSum.ProcessLasso.Beta.installer.yaml

* Updating manifests/b/BitSum/ProcessLasso/Beta/12.4.3.13/BitSum.ProcessLasso.Beta.locale.en-US.yaml

* Updating manifests/b/BitSum/ProcessLasso/Beta/12.4.3.13/BitSum.ProcessLasso.Beta.yaml

wingetbot committed:
Automatic update of softsyst.QIRX4 4.1.3 (#128449)

* Updating manifests/s/softsyst/QIRX4/4.1.3/softsyst.QIRX4.installer.yaml

* Updating manifests/s/softsyst/QIRX4/4.1.3/softsyst.QIRX4.locale.en-US.yaml

* Updating manifests/s/softsyst/QIRX4/4.1.3/softsyst.QIRX4.yaml

wingetbot committed:
Automatic deletion of spacedrive.Spacedrive 0.1.2 (#128450)

* Updating manifests/s/spacedrive/Spacedrive/0.1.2/spacedrive.Spacedrive.installer.yaml

* Updating manifests/s/spacedrive/Spacedrive/0.1.2/spacedrive.Spacedrive.locale.en-US.yaml

* Updating manifests/s/spacedrive/Spacedrive/0.1.2/spacedrive.Spacedrive.yaml

wingetbot committed:
Automatic deletion of Mobirise.Mobirise 5.8.14 (#128452)

* Updating manifests/m/Mobirise/Mobirise/5.8.14/Mobirise.Mobirise.installer.yaml

* Updating manifests/m/Mobirise/Mobirise/5.8.14/Mobirise.Mobirise.locale.en-US.yaml

* Updating manifests/m/Mobirise/Mobirise/5.8.14/Mobirise.Mobirise.yaml

wingetbot committed:
Automatic deletion of BitSum.ProcessLasso.Beta 12.4.3.7 (#128453)

* Updating manifests/b/BitSum/ProcessLasso/Beta/12.4.3.7/BitSum.ProcessLasso.Beta.installer.yaml

* Updating manifests/b/BitSum/ProcessLasso/Beta/12.4.3.7/BitSum.ProcessLasso.Beta.locale.en-US.yaml

* Updating manifests/b/BitSum/ProcessLasso/Beta/12.4.3.7/BitSum.ProcessLasso.Beta.yaml

wingetbot committed:
Automatic deletion of spacedrive.Spacedrive 0.1.3 (#128454)

* Updating manifests/s/spacedrive/Spacedrive/0.1.3/spacedrive.Spacedrive.installer.yaml

* Updating manifests/s/spacedrive/Spacedrive/0.1.3/spacedrive.Spacedrive.locale.en-US.yaml

* Updating manifests/s/spacedrive/Spacedrive/0.1.3/spacedrive.Spacedrive.yaml

wingetbot committed:
Automatic deletion of BitSum.ProcessLasso.Beta 12.4.3.9 (#128456)

* Updating manifests/b/BitSum/ProcessLasso/Beta/12.4.3.9/BitSum.ProcessLasso.Beta.installer.yaml

* Updating manifests/b/BitSum/ProcessLasso/Beta/12.4.3.9/BitSum.ProcessLasso.Beta.locale.en-US.yaml

* Updating manifests/b/BitSum/ProcessLasso/Beta/12.4.3.9/BitSum.ProcessLasso.Beta.yaml

wingetbot committed:
Automatic deletion of MTSD.AllDup 4.5.54 (#128457)

* Updating manifests/m/MTSD/AllDup/4.5.54/MTSD.AllDup.installer.yaml

* Updating manifests/m/MTSD/AllDup/4.5.54/MTSD.AllDup.locale.en-US.yaml

* Updating manifests/m/MTSD/AllDup/4.5.54/MTSD.AllDup.yaml

SpecterShell committed:
New version: Tencent.Weiyun version 5.2.1394 (#127371)

albertopasqualetto committed:
Flutter (#126336)

* Flutter

* Fixed name

* Update manifests/g/Google/Flutter/3.13.9/Google.Flutter.installer.yaml

Co-authored-by: Stephen Gillie <114952565+stephengillie@users.noreply.github.com>

* Flutter 3.16.0

* removed 3.13.9 version

---------

Co-authored-by: Stephen Gillie <114952565+stephengillie@users.noreply.github.com>

Yin-ZP committed:
Live777 (#127396)

* Submitting ContosoApp version 1.0.0

* Submitting ContosoApp version 1.0.0

* Submitting ContosoApp version 1.0.0

* Submitting ContosoApp version 1.0.0

* Submitting ContosoApp version 1.0.0

* Submitting ContosoApp version 1.0.0

* Submitting ContosoApp version 1.0.0

* Submitting ContosoApp version 1.0.0

* Submitting ContosoApp version 1.0.0

* Submitting ContosoApp version 1.0.0

* Update binbat.live777.locale.en-US.yaml

GiulianoSpaghetti committed:
New version: GiulioSorrentino.wxBriscola version 0.6.3 (#127750)

* New version: GiulioSorrentino.wxBriscola version 0.6.3

* Update GiulioSorrentino.wxBriscola.installer.yaml

BrettMayson committed:
New version: BrettMayson.HEMTT version 1.10.2 (#127871)

wingetbot committed:
Automatic update of FireAlpaca.FireAlpaca from 2.11.13.0 to 2.11.14.0 (#128436)

* Creating manifests/f/FireAlpaca/FireAlpaca/2.11.14.0/FireAlpaca.FireAlpaca.installer.yaml

* Creating manifests/f/FireAlpaca/FireAlpaca/2.11.14.0/FireAlpaca.FireAlpaca.locale.en-US.yaml

* Creating manifests/f/FireAlpaca/FireAlpaca/2.11.14.0/FireAlpaca.FireAlpaca.yaml

* Updating manifests/f/FireAlpaca/FireAlpaca/2.11.14.0/FireAlpaca.FireAlpaca.installer.yaml

* Updating manifests/f/FireAlpaca/FireAlpaca/2.11.14.0/FireAlpaca.FireAlpaca.locale.en-US.yaml

* Updating manifests/f/FireAlpaca/FireAlpaca/2.11.14.0/FireAlpaca.FireAlpaca.yaml

b1n4ryj4n committed:
New version: NitrokeyGmbH.Nitrokey-App version 2.1.4 (#128426)

RadikaRules committed:
pixivInc.VRoidStudio version 1.24.1 (#128427)

wingetbot committed:
Automatic update of Soundly.Soundly from 2023.11.0.6 to 2023.11.0.9 (#128451)

* Creating manifests/s/Soundly/Soundly/2023.11.0.9/Soundly.Soundly.installer.yaml

* Creating manifests/s/Soundly/Soundly/2023.11.0.9/Soundly.Soundly.locale.en-US.yaml

* Creating manifests/s/Soundly/Soundly/2023.11.0.9/Soundly.Soundly.yaml

* Updating manifests/s/Soundly/Soundly/2023.11.0.9/Soundly.Soundly.installer.yaml

* Updating manifests/s/Soundly/Soundly/2023.11.0.9/Soundly.Soundly.locale.en-US.yaml

* Updating manifests/s/Soundly/Soundly/2023.11.0.9/Soundly.Soundly.yaml

wingetbot committed:
Automatic update of BitSum.ProcessLasso.Beta from 12.4.3.9 to 12.4.3.13 (#128455)

* Creating manifests/b/BitSum/ProcessLasso/Beta/12.4.3.13/BitSum.ProcessLasso.Beta.installer.yaml

* Creating manifests/b/BitSum/ProcessLasso/Beta/12.4.3.13/BitSum.ProcessLasso.Beta.locale.en-US.yaml

* Creating manifests/b/BitSum/ProcessLasso/Beta/12.4.3.13/BitSum.ProcessLasso.Beta.yaml

* Updating manifests/b/BitSum/ProcessLasso/Beta/12.4.3.13/BitSum.ProcessLasso.Beta.installer.yaml

* Updating manifests/b/BitSum/ProcessLasso/Beta/12.4.3.13/BitSum.ProcessLasso.Beta.locale.en-US.yaml

* Updating manifests/b/BitSum/ProcessLasso/Beta/12.4.3.13/BitSum.ProcessLasso.Beta.yaml

wingetbot committed:
Automatic update of ESET.EndpointSecurity from 10.1.2058.0 to 11.0.2032.0 (#128429)

* Creating manifests/e/ESET/EndpointSecurity/11.0.2032.0/ESET.EndpointSecurity.installer.yaml

* Creating manifests/e/ESET/EndpointSecurity/11.0.2032.0/ESET.EndpointSecurity.locale.en-US.yaml

* Creating manifests/e/ESET/EndpointSecurity/11.0.2032.0/ESET.EndpointSecurity.yaml

* Updating manifests/e/ESET/EndpointSecurity/11.0.2032.0/ESET.EndpointSecurity.installer.yaml

* Updating manifests/e/ESET/EndpointSecurity/11.0.2032.0/ESET.EndpointSecurity.locale.en-US.yaml

* Updating manifests/e/ESET/EndpointSecurity/11.0.2032.0/ESET.EndpointSecurity.yaml

wingetbot committed:
Automatic deletion of softsyst.QIRX4 4.1.2 (#128447)

* Updating manifests/s/softsyst/QIRX4/4.1.2/softsyst.QIRX4.installer.yaml

* Updating manifests/s/softsyst/QIRX4/4.1.2/softsyst.QIRX4.locale.en-US.yaml

* Updating manifests/s/softsyst/QIRX4/4.1.2/softsyst.QIRX4.yaml

bsys6 committed:
Update LibreWolf.LibreWolf to v120.0-2 (#127951)

vedantmgoyal9 committed:
New version: defi.defi version 4.0.6 (#127968)

* New version: defi.defi version 4.0.6

* Change architecture.

---------

Co-authored-by: Stephen Gillie <114952565+stephengillie@users.noreply.github.com>

CodeDoctorDE committed:
New version: LinwoodCloud.Butterfly.Nightly version 2.0.0-beta.16 (#127970)

vedantmgoyal9 committed:
New version: GDevelop.GDevelop version 5.3.181 (#127973)

* New version: GDevelop.GDevelop version 5.3.181

* Change architecture.

---------

Co-authored-by: Stephen Gillie <114952565+stephengillie@users.noreply.github.com>

PckgrBot committed:
New version: Kitware.CMake version 3.27.9 (#128458)

wingetbot committed:
Automatic deletion of CubeCodersLimited.AMPInstanceManager 2.4.6.6 (#128460)

* Updating manifests/c/CubeCodersLimited/AMPInstanceManager/2.4.6.6/CubeCodersLimited.AMPInstanceManager.installer.yaml

* Updating manifests/c/CubeCodersLimited/AMPInstanceManager/2.4.6.6/CubeCodersLimited.AMPInstanceManager.locale.en-US.yaml

* Updating manifests/c/CubeCodersLimited/AMPInstanceManager/2.4.6.6/CubeCodersLimited.AMPInstanceManager.yaml

dehanj committed:
New package: Tillitis.TKeySign version 0.0.8 (#127885)

wingetbot committed:
Updating manifests/t/Tencent/Weiyun/.validation (#128461)

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/smallstep/winget-pkgs?pullRequest=2) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/smallstep/winget-pkgs?pullRequest=2) before we can accept your contribution.<br/>**0** out of **24** committers have signed the CLA.<br/><br/>:x: babblebauble<br/>:x: charmcli<br/>:x: BinBinTsai<br/>:x: adamk-janea<br/>:x: zxdong262<br/>:x: jahvari<br/>:x: wingetbot<br/>:x: Darshit Mistry<br/>:x: twpayne<br/>:x: jay-51<br/>:x: japplis<br/>:x: SpecterShell<br/>:x: CennoxX<br/>:x: fredericboyer<br/>:x: LeonGungadinMogensen<br/>:x: Dragon1573<br/>:x: krystofkrticka<br/>:x: MrHinsh<br/>:x: mdanish-kh<br/>:x: kdpuvvadi<br/>:x: chrisbra<br/>:x: Eddga<br/>:x: newcloudhhc<br/>:x: imfelixlaw<br/><hr/>**Darshit Mistry** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/smallstep/winget-pkgs?pullRequest=2) it.</sub>

nvim-winget committed:
Update version: Neovim.Neovim.Nightly version 0.10.0 (#128489)

ilike2burnthing committed:
New version: Jackett.Jackett version 0.21.1277 (#128490)

wingetbot committed:
Automatic deletion of LabyMediaGmbH.LabyModLauncher 2.0.1 (#128468)

* Updating manifests/l/LabyMediaGmbH/LabyModLauncher/2.0.1/LabyMediaGmbH.LabyModLauncher.installer.yaml

* Updating manifests/l/LabyMediaGmbH/LabyModLauncher/2.0.1/LabyMediaGmbH.LabyModLauncher.locale.en-US.yaml

* Updating manifests/l/LabyMediaGmbH/LabyModLauncher/2.0.1/LabyMediaGmbH.LabyModLauncher.yaml

wingetbot committed:
Automatic deletion of DucFabulous.UltraViewer 6.6.60 (#128467)

* Updating manifests/d/DucFabulous/UltraViewer/6.6.60/DucFabulous.UltraViewer.installer.yaml

* Updating manifests/d/DucFabulous/UltraViewer/6.6.60/DucFabulous.UltraViewer.locale.en-US.yaml

* Updating manifests/d/DucFabulous/UltraViewer/6.6.60/DucFabulous.UltraViewer.yaml

wingetbot committed:
Automatic deletion of Ubisoft.Connect 147.0.10965 (#128464)

* Updating manifests/u/Ubisoft/Connect/147.0.10965/Ubisoft.Connect.installer.yaml

* Updating manifests/u/Ubisoft/Connect/147.0.10965/Ubisoft.Connect.locale.en-US.yaml

* Updating manifests/u/Ubisoft/Connect/147.0.10965/Ubisoft.Connect.yaml

wingetbot committed:
Automatic deletion of TweakingTechnologies.TweakShotScreenCapture 1.0.1.40908 (#128463)

* Updating manifests/t/TweakingTechnologies/TweakShotScreenCapture/1.0.1.40908/TweakingTechnologies.TweakShotScreenCapture.installer.yaml

* Updating manifests/t/TweakingTechnologies/TweakShotScreenCapture/1.0.1.40908/TweakingTechnologies.TweakShotScreenCapture.locale.en-US.yaml

* Updating manifests/t/TweakingTechnologies/TweakShotScreenCapture/1.0.1.40908/TweakingTechnologies.TweakShotScreenCapture.yaml

wingetbot committed:
Automatic deletion of OxygenCloud.odrive 1.0.7226 (#128462)

* Updating manifests/o/OxygenCloud/odrive/1.0.7226/OxygenCloud.odrive.installer.yaml

* Updating manifests/o/OxygenCloud/odrive/1.0.7226/OxygenCloud.odrive.locale.en-US.yaml

* Updating manifests/o/OxygenCloud/odrive/1.0.7226/OxygenCloud.odrive.yaml

JanDeDobbeleer committed:
JanDeDobbeleer.OhMyPosh version 18.27.0 (#128495)

SpecterShell committed:
Update: vivo.OfficeKit version 3.1.3.0 (#128504)

PckgrBot committed:
New version: Amazon.AWSCLI version 2.14.2 (#128470)

SpecterShell committed:
New version: LLVM.LLVM version 17.0.6 (#128517)

russellbanks committed:
New version: Azul.Zulu.8.JRE version 8.66.0.15 (#128525)

russellbanks committed:
New version: Azul.Zulu.8.JRE version 8.68.0.21 (#128527)

russellbanks committed:
New version: Azul.Zulu.8.JRE version 8.68.0.19 (#128526)

russellbanks committed:
New version: Azul.Zulu.8.JRE version 8.70.0.23 (#128528)

russellbanks committed:
New version: Azul.Zulu.8.JRE version 8.72.0.17 (#128529)

russellbanks committed:
New version: Azul.Zulu.8.JRE version 8.74.0.17 (#128530)

russellbanks committed:
New version: Azul.Zulu.8.JDK version 8.66.0.15 (#128519)

russellbanks committed:
New version: Azul.Zulu.8.JDK version 8.74.0.17 (#128524)

russellbanks committed:
New version: Azul.Zulu.8.JDK version 8.70.0.23 (#128522)

russellbanks committed:
New version: Azul.Zulu.8.JDK version 8.68.0.21 (#128521)

russellbanks committed:
New version: Azul.Zulu.8.JDK version 8.72.0.17 (#128523)

russellbanks committed:
New version: Azul.Zulu.8.JDK version 8.68.0.19 (#128520)

russellbanks committed:
New version: Azul.Zulu.11.JDK version 11.68.17 (#128532)

russellbanks committed:
New version: Azul.Zulu.11.JDK version 11.66.19 (#128531)

vedantmgoyal9 committed:
New version: calibre.calibre version 7.1.0 (#127966)

* New version: calibre.calibre version 7.1.0

* Use archive URL

---------

Co-authored-by: Russell Banks <74878137+russellbanks@users.noreply.github.com>

Commit:
Update iCECAT TO VERSION 115.5.0ESR (#127954)

* add files, wingetcreate does not add files, who knew

* fix filename

* Hoping to fix syntax for silent install switch

* Update Icecat.Icecat.installer.yaml

Not sure this will solve. I'm making it up as I go, sorry

* add silentwithprogress...

* Use silent installer

* update shasum

* Trying to fox syntax and adding some details

* fix syntax again

* try different version

* different installer version

* same version for all

* FIX

* Update manifests/i/Icecat/Icecat/115.3.1esr/Icecat.Icecat.installer.yaml

Co-authored-by: Stephen Gillie <114952565+stephengillie@users.noreply.github.com>

* add back the silent switches

* add new version

* Update Icecat.Icecat.locale.en-US.yaml

Add correct licenses

* Update Icecat.Icecat.installer.yaml

Fix typo

* Update Icecat.Icecat.installer.yaml

Fixed Download url server side

* keep new version then

* Try and fix URLs

* Version 115.5

---------

Co-authored-by: g <g@debian-BULLSEYE-live-builder-AMD64>
Co-authored-by: Stephen Gillie <114952565+stephengillie@users.noreply.github.com>

vedantmgoyal9 committed:
New version: OpenWhisperSystems.Signal version 6.39.1 (#127998)

* New version: OpenWhisperSystems.Signal version 6.39.1

* Change architecture.

---------

Co-authored-by: Stephen Gillie <114952565+stephengillie@users.noreply.github.com>

vedantmgoyal9 committed:
New version: OpenWhisperSystems.Signal.Beta version 6.40.0-beta.2 (#127999)

* New version: OpenWhisperSystems.Signal.Beta version 6.40.0-beta.2

* Change architecture.

---------

Co-authored-by: Stephen Gillie <114952565+stephengillie@users.noreply.github.com>

genivia-inc committed:
New version: Genivia.ugrep version 4.3.4 (#128008)

vedantmgoyal9 committed:
New version: Telerik.Fiddler.Everywhere version 5.2.0 (#128011)

* New version: Telerik.Fiddler.Everywhere version 5.2.0

* Change architecture.

---------

Co-authored-by: Stephen Gillie <114952565+stephengillie@users.noreply.github.com>

soredake committed:
Syncplay.Syncplay: add 1.7.1 version (#128055)

vedantmgoyal9 committed:
New version: Telerik.Fiddler.Everywhere.Insiders version 5.2.0-insiders (#128012)

* New version: Telerik.Fiddler.Everywhere.Insiders version 5.2.0-insiders

* Change architecture.

---------

Co-authored-by: Stephen Gillie <114952565+stephengillie@users.noreply.github.com>

lonnywong committed:
Update version: trzsz.tssh version 0.1.14 (#128127)

coolplaylinbot committed:
New version: Genymobile.scrcpy version 2.3 (#128075)

vedantmgoyal9 committed:
New version: dscalzi.HeliosLauncher version 2.0.6 (#128135)

* New version: dscalzi.HeliosLauncher version 2.0.6

* Change architecture.

---------

Co-authored-by: Stephen Gillie <114952565+stephengillie@users.noreply.github.com>

charmcli committed:
New version: charmbracelet.soft-serve 0.7.3 (#128276)

* New version: charmbracelet.soft-serve 0.7.3: add version

* New version: charmbracelet.soft-serve 0.7.3: add installer

* New version: charmbracelet.soft-serve 0.7.3: add locale

---------

Co-authored-by: goreleaserbot <bot@goreleaser.com>

charmcli committed:
New version: charmbracelet.gum 0.12.0 (#128284)

* New version: charmbracelet.gum 0.12.0: add version

* New version: charmbracelet.gum 0.12.0: add installer

* New version: charmbracelet.gum 0.12.0: add locale

---------

Co-authored-by: goreleaserbot <bot@goreleaser.com>

BinBinTsai committed:
New version: Splashtop.SplashtopStreamer.Deployment.MSI version 3.6.2.0 (#128356)

babblebauble committed:
Version Update: Morpheus.Bazarr version 1.4.0 (#128422)

wingetbot committed:
Automatic update of Nlitesoft.NTLite from 2023.11.9477.0 to 2023.11.9514.0 (#128459)

* Creating manifests/n/Nlitesoft/NTLite/2023.11.9514.0/Nlitesoft.NTLite.installer.yaml

* Creating manifests/n/Nlitesoft/NTLite/2023.11.9514.0/Nlitesoft.NTLite.locale.en-US.yaml

* Creating manifests/n/Nlitesoft/NTLite/2023.11.9514.0/Nlitesoft.NTLite.yaml

* Updating manifests/n/Nlitesoft/NTLite/2023.11.9514.0/Nlitesoft.NTLite.installer.yaml

* Updating manifests/n/Nlitesoft/NTLite/2023.11.9514.0/Nlitesoft.NTLite.locale.en-US.yaml

* Updating manifests/n/Nlitesoft/NTLite/2023.11.9514.0/Nlitesoft.NTLite.yaml

jahvari committed:
New package: Microsoft.SaRA version 17.1.268.15 (#127741)

* Microsoft.SaRA version 17.01.0268.015

* Fix for silent installation

Installs SaRA silently.

* Update manifests/m/Microsoft/SaRA/17.01.0268.015/Microsoft.SaRA.installer.yaml

Co-authored-by: Kaleb Luedtke <trenlymc@gmail.com>

* Update manifests/m/Microsoft/SaRA/17.01.0268.015/Microsoft.SaRA.locale.en-US.yaml

Co-authored-by: Kaleb Luedtke <trenlymc@gmail.com>

* Update manifests/m/Microsoft/SaRA/17.01.0268.015/Microsoft.SaRA.locale.en-US.yaml

Co-authored-by: Kaleb Luedtke <trenlymc@gmail.com>

* Update manifests/m/Microsoft/SaRA/17.01.0268.015/Microsoft.SaRA.yaml

Co-authored-by: Kaleb Luedtke <trenlymc@gmail.com>

* Update manifests/m/Microsoft/SaRA/17.01.0268.015/Microsoft.SaRA.locale.en-US.yaml

Co-authored-by: Kaleb Luedtke <trenlymc@gmail.com>

* Update manifests/m/Microsoft/SaRA/17.01.0268.015/Microsoft.SaRA.locale.en-US.yaml

Co-authored-by: Kaleb Luedtke <trenlymc@gmail.com>

* Update manifests/m/Microsoft/SaRA/17.01.0268.015/Microsoft.SaRA.yaml

Co-authored-by: Kaleb Luedtke <trenlymc@gmail.com>

* Update manifests/m/Microsoft/SaRA/17.01.0268.015/Microsoft.SaRA.locale.en-US.yaml

Co-authored-by: Kaleb Luedtke <trenlymc@gmail.com>

* Update manifests/m/Microsoft/SaRA/17.01.0268.015/Microsoft.SaRA.installer.yaml

Co-authored-by: Kaleb Luedtke <trenlymc@gmail.com>

* Fix

* fix the file path

fix for file path

* File path fix

---------

Co-authored-by: Kaleb Luedtke <trenlymc@gmail.com>

adamk-janea committed:
Memurai Developer v4.1.0 (#128475)

Co-authored-by: Adam Kaczmarek <adam@kaczmarek.dev>

adamk-janea committed:
Memurai Developer v4.0.4 (#128474)

Co-authored-by: Adam Kaczmarek <adam@kaczmarek.dev>

zxdong262 committed:
New version: electerm.electerm version 1.37.16 (#128476)

jahvari committed:
Update version: PostgreSQL.PostgreSQL version 16.1 (#128477)

* Update version: PostgreSQL.PostgreSQL version 16.1

Resolve #127503

* New package: Intel.IntelArcBetaGraphics version 31.0.101.4972

Resolves #119812

* Revert "New package: Intel.IntelArcBetaGraphics version 31.0.101.4972"

This reverts commit fe469b4cd68dd68c0842ab649c4abb24f365f7c9.

Commit:
New version: qBittorrent.qBittorrent version 4.6.2 (#128484)

* New version: qBittorrent.qBittorrent version 4.6.2

* Update qBittorrent.qBittorrent.installer.yaml

* Update qBittorrent.qBittorrent.locale.en-US.yaml

* Update qBittorrent.qBittorrent.installer.yaml

twpayne committed:
New version: twpayne.chezmoi 2.42.1 (#128494)

* New version: twpayne.chezmoi 2.42.1: add version

* New version: twpayne.chezmoi 2.42.1: add installer

* New version: twpayne.chezmoi 2.42.1: add locale

SpecterShell committed:
New version: 7S2P.Effie version 3.0.1 (#128496)

SpecterShell committed:
New version: Neemb.Distill version 3.8.8-beta (#128497)

SpecterShell committed:
New version: Bilibili.Livehime version 4.56.0.6136 (#128498)

SpecterShell committed:
New version: Tencent.TencentVideo version 11.83.8358.0 (#128499)

jay-51 committed:
New package: RiotGames.LegendsOfRuneterra.Europe version 64.0.10.5036631 (#128480)

jay-51 committed:
New package: RiotGames.LegendsOfRuneterra.Asia version 49.0.0.4352238 (#128481)

SpecterShell committed:
New version: TominLab.WonderPen version 2.4.1 (#128502)

SpecterShell committed:
New version: Hitencent.JisuPDF version 3.0.0.3021 (#128505)

SpecterShell committed:
New version: Axure.AxureRP.10 version 10.0.0.3912 (#128506)

* New version: Axure.AxureRP.10 version 10.0.0.3912

* Remove Scope

SpecterShell committed:
New version: Wondershare.PDFelement.10 version 10.1.9 (#128507)

SpecterShell committed:
New version: THS.THS version 9.20.71 (#128508)

SpecterShell committed:
New version: EdrawSoft.EdrawMax.CN version 13.0.1.1053 (#128509)

SpecterShell committed:
New version: Baidu.BaiduWenku version 2.0.1 (#128511)

SpecterShell committed:
New version: Neemb.Distill version 3.8.7-beta (#128510)

SpecterShell committed:
New version: 5E.5EClient version 7.1.31 (#128512)

SpecterShell committed:
New version: Logseq.Logseq version 0.10.0 (#128513)

jay-51 committed:
New package: RiotGames.LegendsOfRuneterra.Americas version 64.0.10.5036631 (#128478)

SpecterShell committed:
New version: Seewo.EasiNote version 5.2.3.5015 (#128514)

japplis committed:
Applet Runner Pro version 2.7 (#128515)

MrHinsh committed:
New version: nkdAgility.AzureDevOpsMigrationTools version 14.3.4 (#128518)

SpecterShell committed:
New version: 7S2P.Effie.CN version 3.0.9 (#128516)

jay-51 committed:
New package: Cisco.Webex version 3.9.0.27956 (#128536)

CennoxX committed:
New version: BurntSushi.ripgrep.MSVC version 14.0.3 (#128535)

fredericboyer committed:
Update InstallerSha256 for Ngrok 3.3.5 installers (#128538)

CennoxX committed:
New version: BurntSushi.ripgrep.GNU version 14.0.3 (#128534)

LeonGungadinMogensen committed:
New version: Toit.Jaguar version 1.22.2 (#128543)

Dragon1573 committed:
New version: Sonosaurus.SonoBus version 1.7.0 (#128544)

krystofkrticka committed:
New version: Datronicsoft.SpacedeskDriver.Server version 1.0.77.0 (#128545)

imfelixlaw committed:
New version: Microsoft.Skype version 8.109.0.209 (#128548)

chrisbra committed:
New version: vim.vim.nightly version 9.0.2136 (#128466)

adamk-janea committed:
Memurai Developer v3.1.7 (#128473)

Co-authored-by: Adam Kaczmarek <adam@kaczmarek.dev>

SpecterShell committed:
Update: Wondershare.Recoverit version 12.0.19.5 (#128500)

newcloudhhc committed:
New package: NewCloud WakeOnLan Ver 1.0.0 (#128546)

Co-authored-by: haohuancheng@163.com <haohuancheng@126.com>

jay-51 committed:
New package: RiotGames.LegendsOfRuneterra.SEA version 64.0.10.5036631 (#128482)

SpecterShell committed:
New version: NetEase.QIYU version 2.4.9 (#128501)

kdpuvvadi committed:
New version: Hashicorp.Terraform version 1.6.5 (#128533)

mdanish-kh committed:
Remove Validation-Forbidden-URL-Error on pipeline re-run (#128058)

Eddga committed:
REALiX.HWiNFO version 7.66 (#128325)
